book/src/12_tracked.md

```diff
@@ -0,0 +1,67 @@
+# `FlaggedStorage` and modification events
+
+In most games you will have many entities, but from frame to frame only a few of them will 
+actually need to have components updated.
```

Comment:
For me it's bit unclear what updating components means here: Maybe add some examples on it.
You could also mention keeping some external data structure on sync with changes.